src/components/divider/index.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Divider from ".";
+
+<Meta title="Components/Divider" component={Divider} />
+
+# Divider
+
+Divider component renders hr HTML tag. Using the hr tag, an HTML page can define a theme break (e.g. a shift of topic).
+The hr element, which is used to divide text or indicate a change in an HTML page, is typically presented as a horizontal rule, [see MDN doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr).
+Divider component also expects _assistiveHidden_ prop that adds _aria-hidden_ attribute that can be used to hide non-interactive content from the accessibility API [see MDN doc](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#:~:text=The%20aria%2Dhidden%20attribute%20can,children%20from%20the%20accessibility%20tree)
+
+# Not a Divider
+
+Do not use Divider when the quotation is not a break in the paragraph level of the article’s flow (e.g. Article body block's pull quote)
+
```

Comment:
```suggestion
The `<Divider>` components renders an `<hr>` tag. From a visual perspective, the `<hr>` tag separates content depending on theme styles specified. For example, you may want a blue horizontal line dividing your text. From an accessibility perspective, the `<hr>` tag communicates a "paragraph-level thematic break," according to the [HTML Spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-hr-element). For example, if you're describing different types of sandwiches, a peanut butter and jelly sandwich would need to be divided semantically before talking about a ham and cheese sandwich. 

Yet there may be instances when a custom block developer wants to show only a visual divider without communicating a shift of topic. For example, a divider may be visually useful for separating different items in a list of Top 10 Most Read Stories. Yet the stories are not paragraph-level shifts in topic: they are section-level shifts of topic. In this instance, there's a prop called `assistiveHidden` that takes in boolean. If `true`, then the visual divider will not communicate a content shift via the use of `aria-hidden`. By default, `assistive-hidden` is false.

`aria-hidden="true"` communicates to people who use screen-readers that there is not a shift of paragraph-level topic due to the `<hr>` tag. As a result, the component can look like a divider but not act like one.

Reply:
Thanks @JackHowa . updated it in [135fd7b](https://github.com/WPMedia/arc-themes-components/pull/145/commits/135fd7b57c949aee63c38118f555283351b50392)